db/models.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+  return db.query("SELECT * FROM topics;");
```

Comment:
I'd suggest using a `then` here to deconstruct the returned object so that you're only returning the Array itself